.github/workflows/codeql.yml.template

```diff
@@ -1,4 +1,6 @@
 # TODO: Rename this file to codeql.yml and fill in the TODOs
+# Alternatively, enable codeQL default config via GitHub UI and delete this template
+# See: https://docs.github.com/en/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning#configuring-default-setup-for-a-repository
```

Comment:
Do we really want to keep this config, given that CodeQL can be enabled from the settings?

Reply:
Actually I guess not, should I remove the template entirely? I considered leaving it in for advanced usage, but maybe it's really not useful for most cases.

Reply:
Yes, removing sounds like the best option.

Reply:
done in https://github.com/mdn/project-template/pull/22/commits/e3f8b40d0662f95bd5d0c733a93206199c46f209